README.md

```diff
@@ -64,7 +64,8 @@ then go to `http://localhost:8080`.
 ## Usage
 
 - Clone this repo.
-- Run with `cargo run -- test.js` where `test.js` is an existing JS file with any JS valid code.
+- Run `cargo build`
```

Comment:
For the first time, to be on the safer side, I believe we should mention this step.

Reply:
I don't think this is needed, `cargo` should **always** build if you `run` (and it is not already built). If it doesn't, that means it is a bug in `cargo` and should be reported in the `cargo` repo https://github.com/rust-lang/cargo/

Reply:
Note that we should still remove the `cargo build` part, since it's not needed and is adds complexity to the steps.

Reply:
Removed that part.